src/test/java/com/getindata/connectors/http/internal/utils/ConfigUtilsTest.java

```diff
@@ -94,11 +94,11 @@ public void shouldThrowOnInvalidProperty(String invalidProperty) {
     public void flatMapPropertyMap() {
         Map<String, String> propertyMap = Map.of(
             "propertyOne", "val1",
-            "my.propertyTwo", "val2",
-            "my.super.propertyThree", "val3"
+            "propertyTwo", "val2",
+            "propertyThree", "val3"
         );
```

Comment:
after changes in HttpHeaderUtils, the `toHeaderAndValueArray` is no longer responsible to picking up the header name from property name.

It's job now is only convert map to array